src/control/lib/support/log_test.go

```diff
@@ -370,6 +371,57 @@ func TestSupport_rsyncLog(t *testing.T) {
 	}
 }
 
+func TestSupport_customCopy(t *testing.T) {
+	log, buf := logging.NewTestLogger(t.Name())
+	defer test.ShowBufferOnFailure(t, buf)
+	targetTestDir, targetCleanup := test.CreateTestDir(t)
+	defer targetCleanup()
+
+	rsLog := CollectLogsParams{}
+
+	// Get the current PATH environment variable.
+	oldPathEnv := os.Getenv("PATH")
+	defer os.Setenv("PATH", oldPathEnv)
```

Comment:
Seems kind of strange for a unit test to set the path. I'd be inclined to abstract this out and have a fake here.